providers/statsig/pom.xml

```diff
@@ -19,8 +19,9 @@
     <dependencies>
         <dependency>
             <groupId>com.statsig</groupId>
-            <artifactId>serversdk</artifactId>
-            <version>1.18.1</version>
+            <artifactId>javacore</artifactId>
+            <version>0.12.1</version>
+            <classifier>uber</classifier>
```

Comment:
Why is this uber?  Does this include all of the transitive compiled versions for various platforms?